server/src/main/java/org/apache/iotdb/db/mpp/plan/planner/LogicalPlanBuilder.java

```diff
@@ -607,20 +607,31 @@ public LogicalPlanBuilder planLevelTimeSeriesCountSource(
     return this;
   }
 
-  public LogicalPlanBuilder planChildPathsSchemaSource(PartialPath partialPath) {
-    this.root = new ChildPathsSchemaScanNode(context.getQueryId().genPlanNodeId(), partialPath);
+  public LogicalPlanBuilder planChildPathsSchemaSource(PartialPath partialPath, Integer level) {
```

Comment:
Rename this to planNodePathsSchemaSource, or anything without child but related to node management

Reply:
fixed.